src/components/Search/Addresses.jsx

```diff
@@ -4,6 +4,9 @@ import Address from './Address.jsx';
 class Addresses extends Component {
   constructor(props) {
     super(props);
+    
+    this.handleAddress = this.props.handleAddress;
```

Comment:
I don't think this line is necessary; the constructor should add the props variables as properties on this.